Document GitHub Status check integration. (#117)
ChromaticHQ/usher: README.md

## Configuration

> - Create a `robo.yml` file in the root of your codebase. `robo.drupal.example.yml`

Author: The order didn't seem like it mattered for these, so I changed them. I can be talked into reverting that though.

is provided as a starting point for Drupal projects.
- Create a `.sites.config.yml` file in the root of your codebase. See
`.sites.config.example.yml` for reference on what can/should be configured.
- Add the following to your repo's `composer.json` "scripts" section so that you can call robo easily with `composer robo`:
  ```json
  "scripts": {
    "robo": "robo --ansi"
  }
  ```

> - If you are overriding any commands, add the following to your repo's `composer.json` file.

Reviewer: ```suggestion
- If you are overriding any commands, add the following to your repo's `composer.json` file. Be sure to specify your own namespace prefix in place of `YOURPREFIX`.
```

  ```json
  "autoload": {
  "psr-4": {

>       "BPRobo\\": "robo/src"

Reviewer: This and the reference to BPRobo below should be removed. This is going to depend on whatever namespace the consumer is setting.

Author: Ah, yes, I missed that initial in there!

Should it be `ExampleRobo` or something generic then?

Reviewer: ```suggestion
      "YOURPREFIXRobo\\": "robo/src"
```

    }
  }
  ```
  Then place the commands in `robo/src/Robo/Plugin/Commands` with the

>   `BPRobo\Robo\Plugin\Commands` namespace.

Reviewer: ```suggestion
  `YOURPREFIXRobo\Robo\Plugin\Commands` namespace.
```

> - If you are using any of the commands that use the [`GitHubStatusTrait`](src/Robo/Plugin/Traits/GitHubStatusTrait.php), a personal access token must be configured in GitHub.

Reviewer: ```suggestion
- If you are using any of the commands that use the [`GitHubStatusTrait`](src/Robo/Plugin/Traits/GitHubStatusTrait.php) to set status checks on your pull requests, a personal access token must be created in GitHub.
```

   1. Create a [personal access token](https://github.com/settings/tokens/new) in GitHub.

>    1. Create an environment variable named `GITHUB_ACCESS_TOKEN` with the token value.

Reviewer: ```suggestion
   1. Create an environment variable named `GITHUB_ACCESS_TOKEN` with the token value in the environment where the checks will run.
```
